src/app/containers/App/App.test.jsx

```diff
@@ -121,8 +127,10 @@ describe('App', () => {
             2,
             [],
             {
+              assertUri: undefined,
```

Comment:
This doesn't seem to be breaking anything that was already in this test or any other functionality with the various `setState`s in `App.jsx` but this can be further looked into when refactoring.